commit e97d9ed9a9d71abe285f9f91cfce9fe0f0f72298
Author: Cyril Brulebois <kibi@debian.org>
Date:   Mon Oct 11 00:47:26 2010 +0200

    Revert "dix: don't create core motion events for non-x/y valuators."
    
    This reverts commit d4ef63f602325a9920dc1cbf64e3969dfa394d5f.

diff --git a/dix/eventconvert.c b/dix/eventconvert.c
index 0f747c1..4e3de0b 100644
--- a/dix/eventconvert.c
+++ b/dix/eventconvert.c
@@ -102,15 +102,6 @@ EventToCore(InternalEvent *event, xEvent *core)
     switch(event->any.type)
     {
         case ET_Motion:
-            {
-                DeviceEvent *e = &event->device_event;
-                /* Don't create core motion event if neither x nor y are
-                 * present */
-                if (!BitIsOn(e->valuators.mask, 0) &&
-                    !BitIsOn(e->valuators.mask, 1))
-                    return BadMatch;
-            }
-            /* fallthrough */
         case ET_ButtonPress:
         case ET_ButtonRelease:
         case ET_KeyPress:
